### PR TITLE
Improved UI/UX of drawing app

### PR DESCRIPTION
**Pull request by amrit3701**

### CODE REVIEW

File: Sim_site/drawing_freecad/static/materialize [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../libraries/materialize
```
Comment (amarjeetkapoor1) [inline Sim_site/drawing_freecad/static/materialize:1]: Can't this be specified into the setting.py directly.

**Review by amarjeetkapoor1 — COMMENTED** [review]
What is use of show.html and why is it so big in size?

**Comment by amrit3701** [comment]
On Fri, Jan 13, 2017 at 7:25 PM, amarjeet singh kapoor
<notifications@github.com> wrote:
>
> What is use of show.html and why is it so big in size?

It is the output in the form of WebGL format.

-- 
Amritpal Singh
https://amritpals.com
https://github.com/amrit3701

**Comment by amrit3701** [comment]
On Fri, Jan 13, 2017 at 7:18 PM, amarjeet singh kapoor
<notifications@github.com> wrote:
>
> > @@ -0,0 +1 @@
> +../../../libraries/materialize
>
> Can't this be specified into the setting.py directly.

Maybe yes, but according to me it doesn't nice because the scope of
materialize is outside to the Django app.


-- 
Amritpal Singh
https://amritpals.com
https://github.com/amrit3701

**Comment by amarjeetkapoor1** [comment]
You are just specifying the path in Django Settings. So, It's fine and settings.py is there for this purpose only.

**Comment by amarjeetkapoor1** [comment]
You didn't specify the use of show.html. You just specified the source for where that have orignated. If that file is the output of something then it would be created at compile time instead of adding it in the source code.

**Pull request closed without merging**